addons/website/static/src/scss/website.scss

```diff
@@ -247,7 +247,16 @@ $-seen-urls: ();
     }
 }
 
-.navbar-light {
+// With the introduction of the SalesFour template comes two background color controllers.
+// Using only `.navbar-light` creates a conflict for this template.
+// But changing the selector breaks the color application for the default style.
+// The solution is to manipulate the selector according to the chosen header.
+$o_navbarlight_selector:'.navbar-light';
+@if o-website-value('header-template') == 'salesFour' {
+    $o_navbarlight_selector: '.navbar-light:not(.o_header_salesFour_top_part)';
```

Comment:
Rather than creating an exception for a template's component specifically, I'd prefer to don't trigger these rules for navs that contain a generic class, eg `.navbar-light:not(.o-navbar-light-static)`.

Reply:
That's what I did earlier, but as I wrote in the comment above these lines, changing this selector globally breaks the default `nav-link` style for the other templates. That's why I created this condition to change this selector only for the template that need it.
![Capture d’écran 2023-04-26 à 08 29 41](https://user-images.githubusercontent.com/80679690/234489346-3b32eae9-722f-47a4-b971-a75551da0862.png)

Reply:
The code has been updated and this part is no longer necessary 👍